### ISSUE TITLE
Documentation on how to use ~/.m2 for dependencies

### ISSUE THREAD

**Issue by borissmidt** [VirtusLab/scala-cli#2080]
**Version(s)**
Latest

**Describe what needs to be done and why**
Its not very clear how i can use a dependency which i published to the maven local repository.

Quite often we developed libraries and publish them local maven repository. 

**Is your feature request related to a past ticket or discussion?**
Please mention any other issues or discussions relevant to this one.

**Describe alternatives you've considered**
To `COURSIER_REPOSITORIES=`  i tried adding 
`Dangerous.mavenLocal`
`mavenLocal`
`~/.m2`
`file://~/.m2`
But it seems coursier doesn't document the usage of a the maven local repository. 

**Additional context**
We only use maven/gradle to do the publishing so just chaning to an local ivy isn't trivial.

**Comment by romanowski** [comment]
Hi @borissmidt, indeed our documentation on using repositories is not perfect.

There is no need to add repository as env, we have a [directive](https://scala-cli.virtuslab.org/docs/reference/directives/#repository) and [a command line option ](https://scala-cli.virtuslab.org/docs/reference/cli-options#--repository) for that.

Unfortunately, we do not support predefined local m2 repo but `"file:///~/.m2"` seems to work.

In short, try adding `//> using repository "file:///~/.m2"` to your file (or `project.scala` if you have more then one).

**Comment by borissmidt** [comment]
It seems that ~ doesn't work and i need to use an absolute path

`//> using repository "file:///home/boris/.m2/repository"`
to make it work. But this means that unlike ivy local it isn't very portable between users.

**Comment by romanowski** [comment]
After digging a bit, it seems that we support local m2 as `m2Local` so `//> using repository m2Local` should work across all machines.
